Sample Applications/CustomComboBox/MainWindow.xaml.cs

```diff
@@ -28,5 +29,14 @@ public MainWindow()
         private void Button_Click(object sender, RoutedEventArgs e)
         {
         }
+
+        private void TextBlock_TargetUpdated(object sender, DataTransferEventArgs e)
+        {
+            var listingPeer = ListBoxAutomationPeer.FromElement(sender as TextBlock);
```

Comment:
This code has nothing to do with list, ListBox, etc.   Use a name that's relevant to usage, maybe 'textBlockPeer'.